src/mesh/impls/bout/boutmesh.cxx

```diff
@@ -2210,9 +2210,9 @@ void BoutMesh::addBoundaryRegions() {
   if (xe > xend)
     xe = xend;
 
-  addRegion3D("RGN_LOWER_Y", Region<Ind3D>(xs, xe, 0, ystart-1, 0, LocalNz-1,
+  addRegion3D("RGN_LOWER_Y", Region<Ind3D>(xs, xe, yend, LocalNy-1, 0, LocalNz-1,
```

Comment:
I think `LOWER_Y` still be `0, ystart-1`.

Should be here that gets changed to `yend+1, LocalNy-1`:
https://github.com/boutproject/BOUT-dev/blob/58fc47aa98554f60d714b6391ffad56d3045727b/src/mesh/impls/bout/boutmesh.cxx#L2285-L2288

Reply:
d'oh! Thanks! Too much very similar code!